.github/CODEOWNERS

```
@@ -628,7 +628,7 @@
 # ServiceOwners:                                                   @kpiteira
 
 # PRLabel: %Monitor
-/sdk/monitor/ci.yml                                                @nisha-bhatia @JoshLove-msft @Azure/azure-sdk-write-monitor-data-plane
+/sdk/monitor/                                                      @nisha-bhatia @JoshLove-msft @Azure/azure-sdk-write-monitor-data-plane
```

```suggestion
/sdk/monitor/*                                                     @nisha-bhatia @JoshLove-msft @Azure/azure-sdk-write-monitor-data-plane
```

Just keep in mind that matches go from bottom-to-top, so you'll want to ensure that this does not move below the other Monitor entries.